src/core/ext/filters/client_channel/lb_policy/xds/xds_override_host.cc

```diff
@@ -239,15 +302,21 @@ class XdsOverrideHostLb : public LoadBalancingPolicy {
 
   RefCountedPtr<SubchannelWrapper> LookupSubchannel(absl::string_view address);
 
-  void UpdateAddressMap(const absl::StatusOr<ServerAddressList>& addresses);
+  absl::StatusOr<ServerAddressList> UpdateAddressMap(
+      absl::StatusOr<ServerAddressList> addresses);
 
   RefCountedPtr<SubchannelWrapper> AdoptSubchannel(
       ServerAddress address, RefCountedPtr<SubchannelInterface> subchannel);
 
   void ResetSubchannel(absl::string_view key, SubchannelWrapper* subchannel);
```

Comment:
Suggest renaming this to `UnsetSubchannel()` as well.

Reply:
Done